appcelerator/titanium_mobile
# feat(android): add "google-services.json" support

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27745

**Test:**
1. Download the [kitchensink-v2](https://github.com/appcelerator/kitchensink-v2) app.
2. Build and run on Android.
3. Verify that the app builds and runs without issue.
4. Download [google-services.json](https://axwaysoftware-my.sharepoint.com/:u:/g/personal/jquick_axway_com/EU-gKkYY4wRFpVN4mrOcr3UBI8Ci9-trkF6V8ZdzmAfiVA?e=gMH5Ut). _(Only Axway employees can access this.)_
5. Copy the `google-services.json` file to: `./app/platform/android`
6. Build and run on Android.
7. In the log, verify that you see the following message.
```
[INFO]  [GRADLE] Parsing json file: /Users/<YourFolder>/kitchensink-v2/build/android/app/google-services.json
```
